### ISSUE TITLE
Allow pinch-zooming

### ISSUE THREAD

**Issue by andria-dev** [andria-dev/portfolio#1]
I need to fix my site's `<meta>` tag that specifically denies mobile users the ability to pinch-zoom. The issue is that I have values like `user-scalable=no` and `max-scale` which allows desktop zooming in with <kbd>Ctrl</kbd> + <kbd>+</kbd> but not mobile zooming in.

**Comment by dsoumyajit** [comment]
if you give me permission i can try to fix this issue

**Comment by andria-dev** [comment]
I have now assigned you to this issue @pioneiro 👍
If you have any questions please feel free to ask, I won't always be able to respond immediately though.

**Comment by dsoumyajit** [comment]
thank you and no problem with late reptiles

**Comment by dsoumyajit** [comment]
Hey Chris, sorry I can't get time to work on the issue.

I tried to find some ways to fix that in the first few days but failed. I
tried to add some CSS for the viewport, added specific META for restricting
scrolling but eventually nothing worked.I was also busy for the last couple
of days due to some family stuff. I look forward for some advice from you
about the issue that I couldn't resolve.

Regards and Apologies
Soumyajit Das

**Comment by andria-dev** [comment]
You're okay, I think the goal just got miscommunicated. I'm just trying

to allow pinch-zooming on my site and pinch-zooming is currently
restricted by some meta tags on the site.

If you want to try again just let me know and I can assign it to you
again. Until then, this issue is open for anyone. If not claimed, I'll eventually get to it.

**Comment by pioneiro** [comment]
Oh sorry, I thought to restrict pinch zooming on both desktop and mobile. I can help it to allow pinch zooming.

And sorry due to some family stuff I was not able to give much time for few days.

**Comment by dsoumyajit** [comment]
> You're okay, I think the goal just got miscommunicated. I'm just trying
> 
> to allow pinch-zooming on my site and pinch-zooming is currently
> restricted by some meta tags on the site.
> 
> If you want to try again just let me know and I can assign it to you
> again. Until then, this issue is open for anyone. If not claimed, I'll eventually get to it.

Oh sorry, I thought to restrict pinch zooming on both desktop and mobile. I can help it to allow pinch zooming.

And sorry due to some family stuff I was not able to give much time for few days.

**Comment by dsoumyajit** [comment]
hi, I think I was able to fix the issue but don't know how to push these updates. Can you please help me with that?

**Comment by andria-dev** [comment]
Hi @pioneiro, you'll need to create a **Pull Request**, or for short, PR. You'll need to do the following steps to accomplish this.

1. **Create a fork** of this repository if you haven't already.
2. Ensure that your **local clone's upstream is set to your fork**.
3. **Push your updates** to a new branch on your fork — preferably named after what the changes are about.
4. Visit this repository's homepage on GitHub and it should prompt you to create a PR. If it doesn't, you can visit the Pull Requests tab and click the _"New pull request"_ button.

In the PR, you should give it an accurate title and a succinct description of the changes you've made and say something like "Fixes #<Issue Number>" where the issue number is the number of this issue. The "Fixes..." part allows GitHub to automatically close this issue once your PR is merged 👍

**Comment by dsoumyajit** [comment]
Hello @ChrisBrownie55, I created the pull request. Please review it and let me know about the updates. And apologies for being late about the updates.

**Comment by andria-dev** [comment]
@pioneiro I reviewed the PR and there are just a few changes that I need to you to make before I can merge it. Good work 👍

**Comment by dsoumyajit** [comment]
Okay I'll look into it let you know as soon as possible.
